modules/telco-core-openshift-data-foundation.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/telco_core_ref_design_specs/telco-core-rds.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="telco-core-openshift-data-foundation_{context}"]
+= Red Hat OpenShift Data Foundation
+
+New in this release::
+* No reference design updates in this release
+
+Description::
+{rh-storage-first} is a software-defined storage service for containers.
+For telco core clusters, storage support is provided by {rh-storage} storage services running externally to the application workload cluster.
+{rh-storage} supports separation of storage traffic using secondary CNI networks.
+
+Limits and requirements::
+* In an IPv4/IPv6 dual-stack networking environment, {rh-storage} uses IPv4 addressing.
+For more information, see "Support OpenShift dual stack with {rh-storage} using IPv4".
```

Comment:
There is no xref for "Support OpenShift dual stack with {rh-storage} using IPv4".